From c140e6aea93f8873b92da08d2de97c34b021001f Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Tue, 6 Mar 2018 16:33:09 +0800
Subject: [PATCH 3451/5242] MLK-17698-5 thermal: imx_sc: add PMIC thermal
 sensor for i.MX8QM

commit  2725fbc3a2e6b6f40aca818eb4261c16d78e041c from
https://source.codeaurora.org/external/imx/linux-imx.git

Remove unused thermal sensors and add PMIC thermal sensors
for i.MX8QM.

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Reviewed-by: Bai Ping <ping.bai@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/thermal/imx_sc_thermal.c |    3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/drivers/thermal/imx_sc_thermal.c b/drivers/thermal/imx_sc_thermal.c
index 5c6353a..9b0ea8d 100644
--- a/drivers/thermal/imx_sc_thermal.c
+++ b/drivers/thermal/imx_sc_thermal.c
@@ -44,8 +44,7 @@ enum imx_thermal_trip {
 
 static const sc_rsrc_t imx8qm_sensor_hw_id[] = {
 	SC_R_A53, SC_R_A72, SC_R_GPU_0_PID0, SC_R_GPU_1_PID0,
-	SC_R_DRC_0, SC_R_DRC_1, SC_R_VPU_PID0, SC_R_PMIC_0,
-	SC_R_PMIC_1, SC_R_PMIC_2,
+	SC_R_DRC_0, SC_R_PMIC_0, SC_R_PMIC_1, SC_R_PMIC_2,
 };
 
 static const sc_rsrc_t imx8qxp_sensor_hw_id[] = {
-- 
1.7.9.5

